src/qibocal/auto/runcard.py

```diff
@@ -36,6 +43,15 @@ def __hash__(self) -> int:
         """Each action is uniquely identified by its id."""
         return hash(self.id)
 
+    def dump(self, path):
```

Comment:
Here and below:
```suggestion
    def dump(self, path: Path):
```
as in the other file.